net/zerotier/src/opnsense/mvc/app/controllers/OPNsense/Zerotier/Api/ZerotierController.php

```diff
@@ -229,4 +230,18 @@ private function toggleZerotierNetwork($networkId, $enabled)
         $action = $enabled ? 'join' : 'leave';
         return trim($backend->configdRun("zerotier $action $networkId"));
     }
+
+    private function toggleZerotierService($enabled)
+    {
+        $backend = new Backend();
+        $backend->configdRun("template reload OPNsense/zerotier");
+        $action = $enabled ? "start" : "stop";
+        return trim($backend->configdRun("zerotier $action"));
+    }
```

Comment:
Ok, this explains it - this function should be renamed to fit what it does. toggle is inverting something

Reply:
Yes, that is true, toggle on or off depending on the value of the variable. I believe the function is appropriately named according to my understanding of the word "toggle"